documentation/docs/pages/contributing/index.md

```diff
@@ -1,6 +1,6 @@
 ## Getting involved in ACCESS-AM3
 
-There are several ways we welcome feedback and contributions on ACCESS-AM3 development. **Contributions from people of all career stages and backgrounds are highly encouraged. Development is led by ACCESS-NRI, the Commonwealth Science and Industrial Research Organisation (CSIRO) and the ARC Centre of Excellence for Weather in the 21st Century (W21C).** We follow an open development workflow where possible building on the work of international modelling groups.
+There are several ways we welcome feedback and contributions on ACCESS-AM3 development. **Contributions from people of all career stages and backgrounds are highly encouraged. Development is led by ACCESS-NRI, the Commonwealth Science and Industrial Research Organisation (CSIRO) and the ARC Centre of Excellence for Weather in the 21st Century (W21C).** We follow an open development workflow where possible, building on the work of international modelling groups.
```

Comment:
```suggestion
There are several ways we welcome feedback and contributions on ACCESS-AM3 development. **Contributions from people of all career stages and backgrounds are highly encouraged. Development is led by ACCESS-NRI, the Commonwealth Science and Industrial Research Organisation (CSIRO) and the ARC Centre of Excellence for Weather in the 21st Century (W21C).** We follow a collaborative development workflow, building on the work of international modelling groups.
```
Considering almost everything is under a proprietary license, it's weird to say "open development"

Reply:
I would change bold text to italics as I personally find it too loud/distracting.